.github/workflows/e2e-evm.yml

```diff
@@ -30,7 +30,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v5
         with:
-          go-version: 1.24
+          go-version: 1.22
```

Comment:
Did something break in go 1.24?

Reply:
Yes, I put a comment about this in the go.mod

Reply:
Fixed in update to geth v1.14, as it has support for Go 1.24. Reverted any Go version changes